src/Components/ServiceNotConfigured/ServiceNotConfigured.js

```diff
@@ -0,0 +1,14 @@
+import { Bullseye, EmptyState, EmptyStateIcon, Title } from '@patternfly/react-core';
+import { WrenchIcon } from '@patternfly/react-icons';
+import React from 'react';
+
+export const ServiceNotConfigured = () => (
+    <Bullseye>
+        <EmptyState style={{ paddingTop: 40 }}>
```

Comment:
I would add the styling to `scss` file instead of direct inline styling.

Reply:
Done!